tailwind.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  content: [
+    "./src/**/*.{js,jsx,ts,tsx}",
+  ],
```

Comment:
@Stevenkwizera06 What does this do?

This should be part of the purge option which you can set like this: 

```
purge: {
    enabled: true,
    content: ["./src/**/*.{js,jsx,ts,tsx}"],
  },
```

Guys make sure that you understand `code` you're seeing online. Don't just copy and paste.